packages/snfoundry/contracts/src/lib.cairo

```diff
@@ -1,5 +1,13 @@
-mod YourContract;
```

Comment:
You contract is not in the [latest stable release ](https://github.com/Scaffold-Stark/speedrunstark/blob/challenge-0-simple-nft/packages/snfoundry/contracts/src/lib.cairo)

I dont expect to see this change

Reply:
Yes, I deleted this since the base-challenge branch has YourContract but as you mentioned for challenge-0 the mod is `YourCollectible`

Reply:
Hi! Dont need to delete a file thats has never been on this challenge-0. As you say I expected to see whats has been changed/improved on `YourCollectible`